src/components/Homepage/Nav/Nav.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { DropdownMenu } from './DropdownMenu.js';
+
+export function Nav(props) {
```

Comment:
My idea is that Nav contains more things (buttons I guess) than just DropdownMenu, so that's why I think it should be a parent component for DropdownMenu as well as the other buttons

Reply:
what buttons for example?

Reply:
Maybe the ones for the forecast options like "Daily", "Weekly", etc.

Reply:
okay